src/providers/Pools/utils.js

```diff
@@ -34,13 +35,21 @@ export const calculateRewardsEarned = async (rewardAddress, account, contractIns
   return totalEarned
 }
 
-export const getTotalStaked = async (account, contractInstance) => {
-  const { balanceOf } = poolMethods
+export const getTotalStaked = async (account, contractInstance, chainId) => {
```

Comment:
These changes are not needed. Use the other `Reader` contract and just use `balanceOf`